packages/odyssey/src/scss/components/_toast.scss

```diff
@@ -0,0 +1,150 @@
+// Copyright 2018-Present Okta, Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.ods-toast--pen {
+  display: grid;
+  position: fixed;
+  z-index: 1;
+  right: $spacing-m;
+  bottom: $spacing-m;
+  grid-gap: $spacing-s 0;
+  grid-template-columns: minmax(min-content, max-content);
+  max-height: 100vh;
+  overflow: visible;
+}
```

Comment:
Looks like `.ods-toast-pen` is the container for stackable toasts. This should probably be outside this scss file. @edburyenegren-okta @arnoldsandoval-okta @jeffbelser-okta @jmelberg-okta @nikhilvenkatraman-okta Shall we discuss on how we going to port these changes over to web component world. That gives a clear picture of how we want to split this scss file.

Reply:
I'd rather hold off on any CSS arch movement until we're past the charter alignment point.

AFAIK, we won't see any new WCs until that happens, correct?

That is, in the future we shouldn't be "porting" WCs, we should be creating them on the same team at the same time.

Until we are there, this feels like the easiest way to keep all relevant pieces together. We're already running into some issues with the CSS file separation and where selectors live today.

Reply:
@arnoldsandoval-okta and I are going to do a little moving of pieces to separate these into two components. Will update in a bit.

Reply:
Toast and Pen are now separate components! Okay with moving this around here because we have some work being done by Pen in the demo; however, I think we need a better distinction for separation. "What's easiest for the ShadowDOM" creates some constraints that may not make sense.

e.g. `ul` and `li` should not be housed separately, nor would `.ods-table--wrapper` - how do we decide when parents and children are separate components vs component parts?